benchto-service/src/main/java/com/teradata/benchto/service/BenchmarkService.java

```diff
@@ -82,12 +83,12 @@ public String startBenchmarkRun(String uniqueName, String name, String sequenceI
 
     @Retryable(value = {TransientDataAccessException.class, DataIntegrityViolationException.class}, maxAttempts = 1)
     @Transactional
-    public void finishBenchmarkRun(String uniqueName, String sequenceId, Status status, List<Measurement> measurements, Map<String, String> attributes)
+    public void finishBenchmarkRun(String uniqueName, String sequenceId, Status status, Optional<Instant> endTime, List<Measurement> measurements, Map<String, String> attributes)
```

Comment:
"remove it" was about `Optional` wrapper, not about the parameter. The parameter is in no way deprecated (it's just introduced and is there to stay).